packages/core/src/amazonqFeatureDev/controllers/chat/controller.ts

```diff
@@ -440,7 +441,7 @@ export class FeatureDevController {
             if (!this.isAmazonQVisible) {
                 const open = 'Open chat'
                 const resp = await vscode.window.showInformationMessage(
-                    'Your code suggestions from Amazon Q are ready to review',
+                    'The Amazon Q Developer Agent for software development has generated code for you to review',
```

Comment:
IMO: this is quite verbose

Reply:
This text is suggested by AWS documentation team [here](https://quip-amazon.com/fA3JAlEEK3ab/QFeatureDev-Errors-string-review#temp:C:SAH3f48f3e68c914276b0c7517c6)